src/index.html

```diff
@@ -11,7 +11,206 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname"> Surname:
+            <input
+            type="text"
+            id="surname"
+            name="surname"
+            required
+            autocomplete="off"
+            >
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd point. Fix this for in all places
```suggestion
            <input
	            type="text"
	            id="surname"
	            name="surname"
	            required
	            autocomplete="off"
            >
```